README.md

```diff
@@ -10,3 +10,24 @@ This repository contains versioned TextField suggestion data for eq-questionnair
 | national-identities.json | List of national identities |
 | passport-countries.json | List of countries for passport questions |
 | religions.json | List of religions |
+
+
+Source data files are provided by the business as multiple column csv files. These can be manually added/updated in this repository at ./source-data
+
+- source files to be provided by the business named for the above datasets
+
+- files to be UTF-8 csv
+
+- any values including commas should be double quoted
+
+- header row to contain comma separated language_codes (i.e `en_gb`, `cy`, `ga`, `eo`)
+
+- data rows to contain comma separated translated lookup values (as to be presented in the lookup lists)
+
+- null values for a given language item can be provided by a non-spaced commas. This will result in that term not being presented as a suggestion for that language
+
+json files can be generated from the source csv files using `./scripts/convert_csv_to_json.py <version_folder>`
+
+- default expected source file location `./source-data`
+
+- default json file output location `./data/<version_folder>`
```

Comment:
I don't understand the change to versioning in the repo. We decided a while ago we would have the latest version in the repo and manage versioning by release tags.

Reply:
I couldn't remember what we'd decided about this. I recalled talking about it for the very first iteration of these lists but also that we would re-visit it as we had newer 'versions' sent to us. Happy to carry on using Github releases if it makes it easier. I've removed all references to version management within the repo.